Documentation/ceph-object-multisite.md

```diff
@@ -0,0 +1,123 @@
+
+## Deleting and Reconfiguring the Object Realm
+
+Changes made to the resource's configuration or deletion of the resource are not reflected on the Ceph cluster.
+
+When the ceph-object-realm resource is deleted or modified, the realm is not deleted from the Ceph cluster. Realm deletion must be done via the toolbox.
+
+### Deleting a Realm
+
+The Rook toolbox can modify the Ceph Multisite state via the radosgw-admin command. 
+
+The following command, run via the toolbox, deletes the realm.
+
+```bash
+$ radosgw-admin realm rm --rgw-realm=realm-a
+```
+
+## Deleting and Reconfiguring the Ceph Object Zone Group
+
+Changes made to the resource's configuration or deletion of the resource are not reflected on the Ceph cluster.
+
+When the ceph-object-zone group resource is deleted or modified, the zone group is not deleted from the Ceph cluster. Zone Group deletion must be done through the toolbox.
+
+### Deleting a Zone Group
+
+The Rook toolbox can modify the Ceph Multisite state via the radosgw-admin command. 
+
+The following command, run via the toolbox, deletes the zone group.
+
+```bash
+$ radosgw-admin zonegroup delete --rgw-realm=realm-a --rgw-zonegroup=zone-group-a
+$ radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-group-a
+```
+
+## Deleting and Reconfiguring the Ceph Object Zone
+
+Changes made to the resource's configuration or deletion of the resource are not reflected on the Ceph cluster.
+
+When the ceph-object-zone resource is deleted or modified, the zone is not deleted from the Ceph cluster. Zone deletion must be done through the toolbox.
+
+### Deleting Zone
+
+The Rook toolbox can modify the Ceph Multisite state via the radosgw-admin command.
+
+There are two scenarios possible when deleting a zone.
+The following commands, run via the toolbox, deletes the zone if there is only one zone in the zone group.
+
+```bash
+$ radosgw-admin zone rm --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
+$ radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
+```
+
+In the other scenario, there are more than one zones in a zone group.
+
+Care must be taken when changing which zone is the master zone.
+
+Please read the following [documentation](https://docs.ceph.com/docs/master/radosgw/multisite/#changing-the-metadata-master-zone) before running the below commands: 
+
+The following commands, run via toolboxes, remove the zone from the zone group first, then delete the zone.
+
+```bash
+$ radosgw-admin zonegroup rm --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
+$ radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
+$ radosgw-admin zone rm --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
+$ radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
+```
+### Changing the Master Zone
```

Comment:
I'd suggest moving the re-configuration section before the deletion section. You generally don't need to modify something after it's deleted. :)